go.mod

```diff
@@ -79,6 +79,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/otiai10/copy v1.2.0
```

Comment:
I had a look at Moby's copy, and it is better due to:
- "deep" copy / follow-symlinks is not implemented or supported at all.
- They use better syscalls for the file copy operation which means it should be faster

Lack of windows support does matter because It is only `pkg/backup/filesystem.go` that uses it and we don't build that for window.

What I find a bigger problem is that if target file exists, it will [abort with error](https://github.com/moby/moby/blob/master/daemon/graphdriver/copy/copy.go#L39). I don't think that is the behavior we want?

Reply:
You make good points. LGTM.